core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -43,10 +43,9 @@ oppia.directive('questionsList', [
       restrict: 'E',
       scope: {},
       bindToController: {
-        getSkill: '&skill',
         getTopicId: '&topicId',
```

Comment:
@sophiewu6 -- I saw the justification you wrote for keeping topic ID here. But if this is just supposed to be showing a list of questions that are in the skill, why is it also handling accept/reject? Can those parts be moved outside this reusable directive into just the topic editor only? It's important to make this directive properly reusable if possible, and independent of its context -- that should be a major part of the point of this PR.

So, maybe split the suggestion threads part out into a separate directive that only exists in the topic viewer? Would that be possible?

/cc @aks681 @nithusha21 who might also want to weigh in here.

Reply:
I do feel it was kind of weird to have accept/reject in questions list, but I was not sure. Might need confirmation from code owners.

Reply:
Hm I thought I responded to this, but please do split it. I'm pretty sure it's separable.

Reply:
Hmmm, I agree. I think we can make this separate from this directive.

Reply:
Yeah, I think it's fine to split the suggestions part to another directive.

Reply:
Hi @nithusha21 @aks681  I couldn't seem to figure out how to find and use the suggestion feature in the topic editor. Could you guide me if you know how it works?